src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlAnyElementAttributes.cs

```diff
@@ -16,7 +17,7 @@ namespace System.Xml.Serialization
     /// </devdoc>
     public class XmlAnyElementAttributes : IList
     {
-        private ArrayList _list;
+        private List<object> _list;
```

Comment:
Sorry, not from Microsoft (I'm the guy who reported the issue) but maybe this could be a `List<XmlAnyElementAttribute>` instead and implement the generic `IList`?